frontend/sentry.client.config.ts

```diff
@@ -2,12 +2,25 @@
 // The config you add here will be used whenever a page is visited.
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
+import { isCurrentProductionDeployment } from '@helpers/vercel-helpers';
 import * as Sentry from '@sentry/nextjs';
 import { BrowserTracing } from '@sentry/browser';
 
 const SENTRY_DSN = process.env.NEXT_PUBLIC_SENTRY_DSN;
 
 Sentry.init({
+   async beforeSend(event) {
+      try {
+         const current_production = await isCurrentProductionDeployment();
+         event.tags = { ...event.tags, current_production };
+      } catch (e) {
+         event.extra = {
+            ...event.extra,
+            'Exception Checking Production Deployment': e,
+         };
```

Comment:
Minor: I would also add a new tag indicating that the `event.extra.'Exception Checking Production Deployment'` exists.

Something like: `SecondaryError: true`

Just makes it easy to search

Reply:
Good idea! https://github.com/iFixit/react-commerce/pull/1958/commits/053774c826fb44d90ffbcad910581ee4c79d346a